pycounter/sushi.py

```diff
@@ -184,14 +184,23 @@ def raw_to_full(raw_report):
     :return: a :class:`pycounter.report.CounterReport`
     """
     # pylint: disable=too-many-statements,too-many-branches,too-many-locals
+
+    # try to remove nasty characters from xml
+    raw_converted = "".join(
+        map(
+            lambda ch: ch if ch.isprintable() else " ",
```

Comment:
.isprintable() seems like the correct method to use here, but I would be afraid of messing up something legitimate. Have you tried a few existing XMLs to see what gets replaced? I know it is extra work, but I think that it could prove useful in the long run.